moto/rds/models.py

```diff
@@ -1912,6 +1921,7 @@ def copy_db_snapshot(
             db_instance=source_snapshot.database,
             db_snapshot_identifier=target_snapshot_identifier,
             tags=tags,
+            original_created_at=source_snapshot.original_created_at,
```

Comment:
Shouldn't this be `orginal_created_at=source_snapshot.created_at`?

Reply:
The `original_created_at` is correct: This field shows when the content was originally created. The `created_at` reflects the time when a particular copy was made.

I just tested this on AWS, taking a snapshot `A`, copying it to `B`, and then copying `B` to `C`. The `OriginalSnapshotCreateTime` is identical for all three of them, and it is equeal to `A`'s `SnapshotCreateTime` value.